MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/usermodel/UserResponse.java

```diff
@@ -0,0 +1,188 @@
+package com.mapbox.mapboxandroiddemo.model.usermodel;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.json.JSONObject;
+
+/**
+ * Created by LangstonSmith on 5/9/17.
+ */
+
+public class UserResponse {
+
+  @SerializedName("authorizations")
```

Comment:
Yeaaa, no reason to have them. I thought I might have to use it but it turned out I didn't. Forgot to change. Removed